imageroot/actions/restore-module/20copyenv

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import json
+import sys
+
+import agent
+
+# get old environment from action
+request = json.load(sys.stdin)
+old_envs = request['environment']
+
+# list of variables to restore
+restore_envs = [
+    'MARIADB_ROOT_PASSWORD',
+    'AMPDBUSER',
+    'AMPDBHOST',
+    'AMPDBNAME',
+    'AMPDBENGINE',
+    'AMPDBPASS',
+    'AMPMGRUSER',
+    'AMPMGRPASS',
+    'APACHE_RUN_USER',
+    'APACHE_RUN_GROUP',
```

Comment:
Probably these vars should not exists at all..

Reply:
I know, but they're there :man_shrugging:

Reply:
@Stell0 can we remove them?

Reply:
yes